spec/models/parsers/cardinal_pbmc_count_parser_spec.rb

```diff
@@ -16,9 +16,7 @@
 # DN871908M:H1	1029	1940000	8.37	74.00%	357	675000	2.35	1386	2610000	8.57
 
 def read_file(filename)
-  content = nil
-  File.open(filename, 'r') { |fd| content = fd.read }
```

Comment:
I'm not quite sure why I ended up in this file.

Reply:
Ahh yes, the well factory had briefly broken. And I was tidying up while I investigated. I then realised I'd be better off fixing factory_bot:lint as a first port of call.